content/chainguard/chainguard-images/reference/node/getting-started-node.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Getting Started with the Node Chainguard Image"
+type: "article"
+description: "Tutorial on how to get started with the Node Chainguard Image"
+date: 2023-02-01T11:07:52+02:00
+lastmod: 2023-02-01T11:07:52+02:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "node-guides"
+weight: 610
+toc: true
+---
+
+The Node Chainguard Image is a distroless container image that has the tooling necessary to build and execute Node applications, including  `npm`.
```

Comment:
Would a link to the reference page make sense here? I don't think we link right to github docs do we?

Reply:
good catch!